L1Trigger/TrackFindingTracklet/interface/Settings.h

```diff
@@ -957,8 +965,10 @@ namespace trklet {
 
     std::string skimfile_{""};  //if not empty events will be written out in ascii format to this file
 
-    double bfield_{3.8112};  //B-field in T
-    double c_{0.299792458};  //speed of light m/ns
+    double bfield_{3.8112};       //B-field in T
+    double c_{0.299792458};       //speed of light m/ns
+    int numrinvbins_{6};          //number of rinv bins to use in DR
```

Comment:
Why are all the new DR specific constants not grouped together, so you can have one comment above them saying for DR? e.g. numrinvbins_ is far from varrinvbins_.

Reply:
`numrinvbins_` has been deleted, and the DR specific constants have been grouped together with a comment explaining that they are for DR.